Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Memberships.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Know_Your_Nation_Speedy.Models
+{
```

Comment:
rather have all you { on the same line as the functions

namespace Know_Your_Nation_Speedy.Models{
     ......
     ......
}

Reply:
I have experienced some IDEs dont let you choose

Reply:
will edit, sorry it auto next lines on enter, will keep it cleaner

Reply:
@RetroSabian I would say leave it since it will be consistent within your project. Dont try fight it, it will constantly fix itself

Reply:
@CilliersN if you disagree leave the comment if you dont just resolve discussion

Reply:
nope leave it as is. different IDE's

Reply:
I agree with @FredJEhlers on this one